### DISCUSSION TITLE
YARP Chinese document | 中文文档

### DISCUSSION THREAD

**Discussion by Aloento** [dotnet/yarp#2046] [Show and tell] [answered]
https://yarp.aloen.to/articles/index.html

持续更新中，精力有限，欢迎 PR，仅翻译 Articles 部分

> // Auto-translation of the above:
> Continuously updating, limited energy, welcome PR, only translate the Articles part

**Comment by karelz** [comment, marked as answer]
Thank you @Aloento -- looks like you translated a few articles into Chinese. I bet Chinese devs will like it.
Was it manual or automated translation? Should we feature it on YARP page?
Who is owner of the site?

**Comment by Aloento** [comment]
@karelz Thank you!
The owner is me! https://github.com/Aloento/YARP-CN-Docs
The translator is ChatGPT, and I do the retouching and proofreading.
I checked every sentence, so you don't need to worry about quality.
I'm still working on the remaining articles. You can put the link to your website.
